libs/frontend/feature-build/src/components/ComponentTreeItem.vue

```diff
@@ -16,7 +17,6 @@
     @drop.stop="drop"
   >
     <div
-      ref="elementRef"
       class="component-title"
```

Comment:
Hi @sampullman 

I move the `elementRef` of component-tree-item from title element to the whole container (title + children) so that the behavior feels more natural when the bottom of the title element is hovered.

Before:

https://github.com/samatechtw/pubstudio-builder/assets/59676941/15e85312-bbaa-44dd-840e-dc975be3789f

After:

https://github.com/samatechtw/pubstudio-builder/assets/59676941/a7c5d79e-9f77-45c9-9912-566c22cc0baf

The bottom border of a component-tree-item is expected to show when the bottom section of the whole container is hovered, instead of when the bottom section of the title element is hovered.

Reply:
Nice!